frontend/src/components/Vendor.vue

```diff
@@ -0,0 +1,83 @@
+<!--
+Copyright (c) 2019 by SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<template>
+  <v-tooltip top>
```

Comment:
What is the value of the tooltip. It shows the same information in a different way.

Reply:
yes on the details page it does not give any benefit, however on the shoot list row it does add some value
we could disable the tooltip on the shoot details page. WDYT?

Reply:
see above

Reply:
✅